### PR TITLE
Fixes the Alert Extended Information widget's script name

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-41628

## Description
Replaced the CortexXDRAdditionalAlertInformationWidget with XCloudAdditionalAlertInformationWidget

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CloudIncidentResponse/Layouts/layoutscontainer-Cloud_Alerts.json [hunk 1]
```diff
@@ -44,192 +44,93 @@
...
                             {
                                 "dropEffect": "move",
-                                "endCol": 2,
-                                "fieldId": "playbookid",
+                                "endCol": 4,
+                                "fieldId": "categoryname",
                                 "height": 26,
-                                "id": "incident-playbookId-field",
-                                "index": 4,
+                                "id": "298513a0-ffa4-11ed-8065-135924776b58",
```
Comment (ssokolovich) [inline Packs/CloudIncidentResponse/Layouts/layoutscontainer-Cloud_Alerts.json:107]: Can this ID be changed?

File: Packs/CloudIncidentResponse/ReleaseNotes/1_0_19.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Layouts
+
+##### Cloud Alerts
+
+- Replaced the CortexXDRAdditionalAlertInformationWidget with XCloudAdditionalAlertInformationWidget
```
Comment (ssokolovich) [inline Packs/CloudIncidentResponse/ReleaseNotes/1_0_19.md:6]:
```suggestion
- Replaced the "CortexXDRAdditionalAlertInformationWidget" with "XCloudAdditionalAlertInformationWidget".
```

File: Packs/CloudIncidentResponse/ReleaseNotes/1_0_19.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+- Replaced 'XDR' fields with common fields
```
Comment (ssokolovich) [inline Packs/CloudIncidentResponse/ReleaseNotes/1_0_19.md:7]:
```suggestion
- Replaced XDR incident fields with common alert fields.
```

File: Packs/CloudIncidentResponse/Layouts/layoutscontainer-Cloud_Alerts.json (hunk 1 above)
Comment (melamedbn) [reply Packs/CloudIncidentResponse/Layouts/layoutscontainer-Cloud_Alerts.json:107]: yea, it's the section ID

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**